webpack.config.js

```diff
@@ -0,0 +1,152 @@
+var path = require('path');
+
+module.exports = (env, argv) => {
+  let devtool = false;
+  if (argv.mode === 'development') {
+    devtool = 'inline-source-map';
+  }
+  console.log(`${argv.mode} build`);
+  const externals = {
+    aframe: {
+      commonjs: 'aframe',
+      commonjs2: 'aframe',
+      amd: 'aframe',
+      root: 'AFRAME' // indicates global variable
+    },
+    three: {
+      commonjs: 'three',
+      commonjs2: 'three',
+      amd: 'three',
+      root: 'THREE' // indicates global variable
+    }
+  };
+  const module = {
+    rules: [
+      {
+        test: /\.worker\.js$/,
```

Comment:
We should check this, the worker-loader should inline the worker inside the bundle but we have worker files next the bundled files.

Reply:
That should be easily doable (https://webpack.js.org/loaders/worker-loader/#inline).

Reply:
I will check this today.

Reply:
I solved, it needs the [inline](https://github.com/webpack-contrib/worker-loader#inline  ) options added no-fallback. Forgot to add this while upgrading the package. We can discuss if it is better the `no-fallback` or `fallback` option, what do you think?